core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -126,9 +126,13 @@ var PreferencesPage = function() {
   };
 
   this.setUserInterests = async function(interests) {
-    await userInterestsInput.click();
+    await action.click('User Interest Input', userInterestsInput);
     for (var i = 0; i < interests.length; i++) {
-      await userInterestsInput.sendKeys(interests[i], protractor.Key.RETURN);
+      await action.sendKeys(
+        'User Interest Input', userInterestsInput, interests[i]);
+      await action.sendKeys(
+        'User Interest Input', userInterestsInput,
+        protractor.Key.RETURN);
       await saveNewChanges('User Interests');
     }
```

Comment:
This is the one which might be causing error

Reply:
When you watch the test run, what happens? For example, if your changes here are the issue, we might expect the test to get stuck adding a user's interest like the `RETURN` key being sent doesn't work